README.md

```diff
@@ -23,3 +26,14 @@ client, err := NewMultiClient(account, "identifier", 3, true, conf)
 client.Send(nkn.NewStringArray("DNS:foo.com"), "Hello world.", nil)
 ```
 
+## Use custom username
+
+Address in a dns TXT record use custom username, like this:
+
+> user1.foo.com TXT nkn=abc...def \
```

Comment:
Should we use `_nkn.user1.foo.com`?

Reply:
I think we should use something like
```
_nkn.foo.com TXT user1@nkn=123abc
_nkn.foo.com TXT user2@nkn=456def
```

Reply:
> I think we should use something like
> 
> ```
> _nkn.foo.com TXT user1@nkn=123abc
> _nkn.foo.com TXT user2@nkn=456def
> ```
> 
> This way we can also support username + subdomain like `user@bar.foo.com`
This way maybe only support 10 record. How about `_nkn.user1.foo.com `?